docs/reference/commandline/node_update.md

```diff
@@ -19,10 +19,31 @@ Update a node
 Options:
       --availability string   Availability of the node (active/pause/drain)
       --help                  Print usage
+      --label-add value       Add/update node labels (key1=value1,key2=value2)
```

Comment:
It's not really related to cobra, we could "make it work" either way. The behaviour is set by the type of flag we use.

`--label-add` is using an `opts.ListOpts` which does not support the comma-separated values. This is the same flag type that `docker run --label` uses.

`--label-rm` is using a `StringSliceVar` flag, which uses the csv parser to support comma separated values, but it doesn't support validators for each item in the list (which `opts.ListOpts` supports).

It sounds like we'd like to just use `opts.ListOpts` for the `--x--rm` options, so that these will be consistent (and also consistent with existing `docker run` flags).